documentation/staging/content/faq/resource-settings.md

```diff
@@ -23,13 +23,49 @@ description: "Tune container memory and CPU usage by configuring Kubernetes reso
 
 #### Introduction
 
-The operator creates a container in its own Pod for each WebLogic Server instance. You can tune container memory and CPU usage by configuring Kubernetes resource requests and limits, and you can tune a WebLogic JVM heap usage using the `USER_MEM_ARGS` environment variable in your Domain YAML file. A resource request sets the minimum amount of a resource that a container requires. A resource limit is the maximum amount of a resource a container is given and prevents a container from using more than its share of a resource. Additionally, resource requests and limits determine a Pod's quality of service.
+The CPU and memory requests and limits for WebLogic server pods usually need to be tuned
+where the optimal values depend on your workload, applications, and the Kubernetes environment.
+Requests and limits should be configured based on the expected traffic during peak usage.
+For example, if you have an application that processes video files,
```

Comment:
I wonder if the applications processing video files is a typical example of compute intensive WebLogic workload. Otherwise we might be able to give some other example such as applications that parse large XML files or other applications that perform intensive computing operations.

Reply:
As far as I know, using WLS to process video files is not a very typical use case.  Perhaps re-word it to be more vague?  Something like, "any workloads that require large amounts of in-memory processing or very CPU intensive calculations"?

Reply:
I've updated the wording, please take a look.